### ISSUE TITLE
Refactor end to end tests to use .atKeys file from secret rather than at_demo_data.dart

### ISSUE THREAD

**Issue by cpswan** [atsign-foundation/at_server#240]
**Is your feature request related to a problem? Please describe.**
The end to end tests presently make use of a modified at_demo_data.dart file that's kept in a GitHub secret and copied into place as part of the GitHub Action run. This file is created by dumping the secrets from a set of .atKeys files and copying those into place - a manual and potentially error prone process. The file also contains invalid CRAM keys, as they need to be there, but CRAM authentication has already taken place to generate .atKeys.

**Describe the solution you'd like**
Rework update_verb_test.dart so that it extracts secrets from a set of .atKeys (named from at_server/at_end2end_test/config/config.yaml) where those .atKeys will be copied into place from GitHub secrets.

**Describe alternatives you've considered**
We could script the process to get from .atKeys to at_demo_data.dart, but that wouldn't deal with the vestigial CRAM keys or use the client SDK mechanisms for working with .atKeys.

**Comment by gkc** [comment]
@cpswan I think you completed this a few months back ... so can we close?

**Comment by cpswan** [comment]
@gkc nope, this hasn't been started. You may be thinking of the work I did to add some new keys to at_demo_data.dart for e2e tests.

**Comment by gkc** [comment]
@cpswan ah yes indeed I was

FYI I am using .atKeys files for the Java e2e tests (which I will be merging just as soon as I can fix up my maven boilerplate to separate unit tests from integration tests)
